tests/monotouch-test/Vision/VNRequestTests.cs

```diff
@@ -136,8 +136,12 @@ public void VNSupportedRevisionsUnsupportedTest ()
 				Assert.That (faceObservation.BoundingBox, Is.EqualTo (rect));
 
 				var recognizedObjectObservation = VNRecognizedObjectObservation.FromBoundingBox (VNRecognizedObjectObservationRequestRevision.Unspecified, rect);
+#if __MACOS__
+				if (TestRuntime.CheckXcodeVersion (11, 0) && !TestRuntime.CheckXcodeVersion (12, 2)) {
```

Comment:
Doesn't `TestRuntime.MinorXcode12APIMismatch` work in this file too?

Reply:
Yep, another commit is coming for the missing cases I have